packages/plugin-ext-vscode/src/node/plugin-vscode-init-fe.ts

```diff
@@ -0,0 +1,42 @@
+/********************************************************************************
+ * Copyright (C) 2022 Arm
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+const _scope = (this as any);
```

Comment:
This file injects `exports` and `require` functionality to support VSCode web extensions as they are built for the commonjs format and `require` vscode for functionality